### PR TITLE
clean up update_position.

### PR DESCRIPTION
**Pull request by ibawt**
cleaned up the update_position.  Added some specs.

@kmcphillips @qq99

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,8 +22,11 @@ gem 'jquery-ui-rails'
 gem 'font-awesome-rails'
 gem 'underscore-rails'
 
+group :test do
+    gem 'webmock', '1.15'
```
Comment (ibawt) [inline Gemfile:26]: this needs to be in test :S

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -130,11 +121,7 @@ def destroy
   private
 
   def needs_refresh?
-    if last_edit = LastEdit.last
-      last_edit.user_id == current_user.id
-    else
-      false
-    end
+    false
```
Comment (ibawt) [inline app/controllers/projects_controller.rb:124]: this is broken and needs to be refactored etc.

**Comment by ibawt** [comment]
I'm thinking that the github client instantiation should be refactored into a method so we're not logging in when we don't have to.

Something like:

``` ruby
def client
  unless @client 
      init client
   end
   @client
end
```

Then we won't get all the vcr's for each action. And we won't rate limit as hard, requests will be faster...I'm going to go do this now I guess lol.

**Comment by karlhungus** [comment]
:tophat:

**Pull request merged**